components/announcement_bar/default_announcement_bar/announcement_bar.tsx

```diff
@@ -112,11 +145,11 @@ export default class AnnouncementBar extends React.PureComponent<Props> {
                 <FormattedMarkdownMessage id={this.props.message}/>
             );
         }
-        const announcementTooltip = (
+        const announcementTooltip = this.state.showTooltip ? (
             <Tooltip id='announcement-bar__tooltip'>
                 {this.props.tooltipMsg ? this.props.tooltipMsg : message}
             </Tooltip>
-        );
+        ) : <></>;
```

Comment:
Nit: Can we have this return `null` so we don't have a `React.Fragment` in the snapshot? It's just a bit cleaner :)

Reply:
Thanks for the comment. In the beginning I tried to pass null but in case of no tooltip is needed, `OverlayTrigger ` component was throwing error because I was passing null for `overlap` prop on [line 167](https://github.com/mattermost/mattermost-webapp/blob/e73b161f528db8f774eb428b28cf36350c85dbec/components/announcement_bar/default_announcement_bar/announcement_bar.tsx#L167)

Reply:
Ahh so it's a type issue. We can leave it for now.